docs/source/reference/api-server/api-server-admin-deploy.rst

```diff
@@ -24,13 +24,15 @@ Prerequisites
 Step 1: Create a namespace for the API server
```

Comment:
Should we also update the places we tell people to use `skypilot-beta` from the s3 link?